src/main/java/weiss/wordle/WordleController.java

```diff
@@ -0,0 +1,98 @@
+package weiss.wordle;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class WordleController {
+
+    int row = 0;
+    int column = 0;
+
+    StringBuilder theWord = new StringBuilder();
+    private WordleGame wordleGame;
+    private WordleDictionary dictionary;
+    private JLabel labels[][];
+    private JButton keyboard[];
+    private JButton enter;
+    private JButton backspace;
+
+
+    public WordleController(WordleGame wordleGame, WordleDictionary dictionary,
+                            JLabel[][] labels, JButton[] keyboard,
+                            JButton enter, JButton backspace) {
+        this.wordleGame = wordleGame;
+        this.dictionary = dictionary;
+        this.labels = labels;
+        this.keyboard = keyboard;
+        this.enter = enter;
+        this.backspace = backspace;
+
+
+    }
+
+    //Called when you type the letter in or press a letter on the onscreen keyboard
+    public void addLetter(String letter) {
+
+        if (labels[row][column].getText().equals("")) {
+            labels[row][column].setText(letter);
+            column++;
+
+            if (column == 5) {
+                column = 0;
+                row++;
+            }
+
+        }
+
+
+    }
+
+    //when you press the enter button
+    public void enterGuess() {
+
+        row--;
+        for (column = 0; column < 5; column++) {
+            theWord.append(labels[row][column].getText());
```

Comment:
Because theWord is only used in this method, make it a local variable not a member variable.